src/System.Net.Security/tests/System.Net.Security.Tests.builds

```diff
@@ -4,20 +4,30 @@
   <ItemGroup>
     <Project Include="FunctionalTests\System.Net.Security.Tests.csproj">
       <OSGroup>Linux</OSGroup>
+      <TargetGroup>netstandard1.7</TargetGroup>
+      <TestTFMs>netcoreapp1.1</TestTFMs>
     </Project>
     <Project Include="FunctionalTests\System.Net.Security.Tests.csproj">
       <OSGroup>OSX</OSGroup>
+      <TargetGroup>netstandard1.7</TargetGroup>
+      <TestTFMs>netcoreapp1.1</TestTFMs>
```

Comment:
One thing I'm missing here is testing netcoreapp1.1 on non-Windows. I'll add another set for netcoreapp, similar to Windows to ensure both variants are tested.